UsefulProteomicsDatabases/DecoyProteinGenerator.cs

```diff
@@ -393,6 +360,36 @@ private static List<Protein> GenerateSlideDecoys(List<Protein> proteins, int max
             return decoyProteins;
         }
 
+        private static Dictionary<int, List<Modification>> SlideProteinSequenceWithMods (char[] sequenceArraySlided, char[] sequenceArrayUnslided, bool initiatorMethionine, int numSlides, Protein protein)
```

Comment:
Would also be good to have a unit test for slide modifications since there is also not one previously existing.